boxwise_flask/resolvers.py

```diff
@@ -55,4 +57,12 @@ def resolve_user(_, info, email):
     return response
 
 
-schema = make_executable_schema(type_defs, query, snake_case_fallback_resolvers)
+@mutation.field("createBox")
+def create_box(_, info, input):
+    response = Stock.create_box(input)
+    return response
+
+
+schema = make_executable_schema(
+    type_defs, [query, mutation], snake_case_fallback_resolvers
```

Comment:
gql(type_defs + query_defs + mutation_defs)
I've described what this means below